packages/expo-updates/ios/EXUpdates/UpdatesUtils.swift

```diff
@@ -96,6 +94,26 @@ public final class UpdatesUtils: NSObject {
     return config.runtimeVersion ?? config.sdkVersion ?? "1"
   }
 
+  internal static func embeddedAssetsMap(withConfig config: UpdatesConfig, database: UpdatesDatabase?) -> [String: Any] {
+    var assetFilesMap: [String: Any] = [:]
+    let embeddedManifest: Update? = EmbeddedAppLoader.embeddedManifest(withConfig: config, database: database)
+    let embeddedAssets = embeddedManifest?.assets() ?? []
+
+    // Prepopulate with embedded assets
+    for asset in embeddedAssets {
+      if let assetKey: String = asset.key {
```

Comment:
optional: no need for explicit type

```suggestion
      if let assetKey = asset.key {
```